bindings/matrix-sdk-ffi/src/ruma.rs

```diff
@@ -277,31 +277,31 @@ impl TryFrom<MessageType> for RumaMessageType {
                     RumaImageMessageEventContent::new(content.body, (*content.source).clone())
                         .info(content.info.map(Into::into).map(Box::new));
                 event_content.formatted = content.formatted.map(Into::into);
-                event_content.filename = content.filename;
+                event_content.filename = Some(content.filename);
```

Comment:
I'm not 100% about these mappings going back to Ruma. Is it important to store the non-computed filename in the FFI type so that we can pass that back here, or is that an unnecessary step?

Reply:
I think this is technically only used for edits and media events can't have them AFAICT, so it might not be needed. However, I don't see the harm in passing it around in case we might need it in the future.

Reply:
I've added it back as `raw_filename`.